src/main/aerospike.cc

```diff
@@ -93,8 +94,8 @@ NAN_METHOD(setDefaultLogging)
 		if (log_from_jsobject(&g_log_info, info[0].As<Object>()) == AS_NODE_PARAM_OK) {
 			if (g_log_info.level < 0) {
 				// common logging does not support log level "OFF"
-				as_log_set_level(AS_LOG_LEVEL_ERROR);
-				as_log_set_callback(NULL);
+				as_log_set_level(AS_LOG_LEVEL_TRACE);
+				as_log_set_callback(as_log_callback_fnct);
```

Comment:
I really think you should revert this change! Else setting the log level to OFF on the JS side, will enable full TRACE level logging in the C client.